nuxeo-coldstorage/src/main/java/org/nuxeo/coldstorage/action/PropagateMoveToColdStorageContentAction.java

```diff
@@ -34,55 +34,59 @@
-import org.nuxeo.ecm.core.api.IdRef;
 import org.nuxeo.ecm.core.bulk.action.computation.AbstractBulkComputation;
 import org.nuxeo.lib.stream.computation.Topology;
 import org.nuxeo.runtime.api.Framework;
 import org.nuxeo.runtime.stream.StreamProcessorTopology;
 
 /**
- * @since 10.10
+ * Bulk action in charge of moving documents referencing a blob shared by another document that was just moved to cold
+ * storage.
+ *
+ * @since 2021.0.0
  */
-public class DeduplicationColdStorageContentActions implements StreamProcessorTopology {
+public class PropagateMoveToColdStorageContentAction implements StreamProcessorTopology {
+
+    private static final Logger log = LogManager.getLogger(PropagateMoveToColdStorageContentAction.class);
 
-    private static final Logger log = LogManager.getLogger(DeduplicationColdStorageContentActions.class);
+    public static final String ACTION_NAME = "propagateMoveToColdStorage";
 
-    public static final String ACTION_NAME = "deduplicationColdStorageContent";
+    public static final String ACTION_FULL_NAME = "bulk/" + ACTION_NAME;
 
     @Override
     public Topology getTopology(Map<String, String> options) {
         return Topology.builder()
-                       .addComputation(DeduplicationColdStorageContentComputation::new, //
-                               List.of(INPUT_1 + ":" + ACTION_NAME, OUTPUT_1 + ":" + STATUS_STREAM))
+                       .addComputation(PropagateMoveToColdStorageContentComputation::new, //
+                               List.of(INPUT_1 + ":" + ACTION_FULL_NAME, OUTPUT_1 + ":" + STATUS_STREAM))
                        .build();
     }
 
-    public static class DeduplicationColdStorageContentComputation extends AbstractBulkComputation {
+    public static class PropagateMoveToColdStorageContentComputation extends AbstractBulkComputation {
 
-        public DeduplicationColdStorageContentComputation() {
-            super(ACTION_NAME);
+        public PropagateMoveToColdStorageContentComputation() {
+            super(ACTION_FULL_NAME);
         }
 
         @Override
         protected void compute(CoreSession session, List<String> ids, Map<String, Serializable> properties) {
-            log.debug("Start computing duplicated ColdStorage content for documents {}", ids);
+            log.debug("Start computing documents of which content has been sent to ColdStorage {}", ids);
             DocumentModelList documents = loadDocuments(session, ids);
 
             ColdStorageService service = Framework.getService(ColdStorageService.class);
 
             for (DocumentModel document : documents) {
                 // Normally it shouldn't be the case
                 if (!document.hasFacet(ColdStorageConstants.COLD_STORAGE_FACET_NAME)) {
-                    DocumentModel documentModel = service.moveToColdStorage(session, document.getRef());
+                    DocumentModel documentModel = service.proceedMoveToColdStorage(session, document.getRef());
```

Comment:
A NuxeoException can be thrown (you fire an event and some custom listener might fail) shouldn't it be caught and count as error in the status?